app/Filament/Resources/EventResource.php

```diff
@@ -70,7 +70,7 @@ public static function infolist(Infolist $infolist): Infolist
 
                         Infolists\Components\TextEntry::make('legacyGame.forumTopic.id')
                             ->label('Forum Topic ID')
-                            ->url(fn (?int $state) => route('forum-topic.show', ['topic' => $state]))
+                            ->url(fn (?int $state) => $state ? route('forum-topic.show', ['topic' => $state]) : null)
```

Comment:
recommend adding `->placeholder('none')` to mimic behavior on hubs:
![image](https://github.com/user-attachments/assets/f41ad35e-7153-4144-bfab-aac671886b16)

Reply:
Updated both infolists in latest to use the "none" placeholder.